makeflow/src/dag_visitors.c

```diff
@@ -521,10 +521,10 @@ void dag_to_cyto(struct dag *d, int condense_display, int change_size)
 	fprintf(cytograph, "\t<att name = \"layoutAlgorithm\" value = \"Grid Layout\" type = \"string\" cy:hidden = \"1\"/>\n");
 
 	if(change_size) {
-		hash_table_firstkey(d->completed_files);
-		while(hash_table_nextkey(d->completed_files, &label, (void **) &name)) {
-			stat(label, &st);
-			average += ((double) st.st_size) / ((double) hash_table_size(d->completed_files));
+		hash_table_firstkey(d->files);
+		while(hash_table_nextkey(d->files, &name, (void **) &f) && dag_file_exists(f)) {
+			stat(name, &st);
```

Comment:
@nhazekam. Let's rename the function to something like dag_file_should_exist. Also, I would like the check on the stat since it can silently mess up the average variable.

Note that on line 689 we do check for the stat to be succeed, so it is not a big deal.